File: notif/createBindings.py
print("+++ Create Bindings.")
import os
from twilio.rest import Client
from datetime import date
# Documentation: https://www.twilio.com/docs/notify/api/bindings
# ------------------------------------------------------------------------------

account_sid = os.environ.get("ACCOUNT_SID")
auth_token = os.environ.get("AUTH_TOKEN")
client = Client(account_sid, auth_token)

notifyServiceSid = os.environ.get("NOTIFY_SERVICE_SID")
phoneNumber1 = os.environ.get("PHONE_NUMBER_1")
phoneNumber2 = os.environ.get("PHONE_NUMBER_2")
phoneNumber3 = os.environ.get("PHONE_NUMBER_3")

binding = client.notify.services(notifyServiceSid).bindings \
            .create(
                identity="peter",
                tag="one",
                binding_type="sms",
                address=phoneNumber1
            )
print("+ Created: " + binding.identity)

binding = client.notify.services(notifyServiceSid).bindings \
            .create(
                identity="paul",
                tag="other",
                binding_type="sms",
                address=phoneNumber2
            )
print("+ Created: " + binding.identity)

binding = client.notify.services(notifyServiceSid).bindings \
            .create(
                identity="mary",
                tag="other",
                binding_type="sms",
                address=phoneNumber3
            )
print("+ Created: " + binding.identity)

print("+++ List Bindings (SID, tag, identity, type, address:phone number).")
bindings = client.notify.services(notifyServiceSid).bindings \
    .list(start_date=date(2015, 8, 25))
for binding in bindings:
    print("+ " + binding.sid + " " + binding.tags[0] + " " + binding.identity + " " + binding.binding_type + " " + binding.address)

# ------------------------------------------------------------------------------
print("+ End of list.")

File: notify/createBindings.js
// -----------------------------------------------------------------------------
console.log("+++ Create a Binding.");

const accountSid = process.env.ACCOUNT_SID;
const authToken = process.env.AUTH_TOKEN;
const client = require('twilio')(accountSid, authToken);

// $ node createBindings.js peter $PHONE_NUMBER1 one
// $ node createBindings.js paul $PHONE_NUMBER2 other
// $ node createBindings.js mary $PHONE_NUMBER3 other
// $ node createBindings.js david $PHONE_NUMBER4
// $ node listBindingsByTag.js
var theIdentity = process.argv[2] || "test1";
var thePhoneNumber = process.argv[3] || "+16505551111";
var theTag = process.argv[4] || "";

console.log("+ Create SMS binding for: " + theIdentity + ", Phone Number = " + thePhoneNumber + ", tag = " + theTag);
console.log("+ NOTIFY_SERVICE_SID: " + process.env.NOTIFY_SERVICE_SID);
client.notify.services(process.env.NOTIFY_SERVICE_SID).bindings
        .create({
            bindingType: 'sms',
            identity: theIdentity,
            address: thePhoneNumber,
            tag: theTag
        })
        .then(
        binding => console.log("+ Created : " + binding.sid)
        ).catch(function (err) {
            if (err) {
                console.error("- Error: ", err.message);
                exit();
        }});

// -----------------------------------------------------------------------------
